test/e2e/testing-manifests/embed.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing_manifests
+
+import (
+	"embed"
+
+	e2etestfiles "k8s.io/kubernetes/test/e2e/framework/testfiles"
+)
+
+//go:embed *
```

Comment:
same question... won't this embed `embed.go`?

Reply:
would it be better to put the embed util in `test/e2e/embed.go` and have it `//go:embed testing-manifests`?

Reply:
> would it be better to put the embed util in test/e2e/embed.go and have it //go:embed testing-manifests?

I did try this approach of having the embed util in a directory above first. The imports of the util functions were looking weird. For example getting the images and fixtures would need an embed util in `test` directory.

Do think having the embed util for testing-manifests in e2e would sound better while the ones for images and fixtures remain inside those directories?

Reply:
I don't feel that strongly about the location, but I do think we should be more specific about what gets embedded